app/helpers/foreman_ansible/ansible_reports_helper.rb

```diff
@@ -28,6 +28,7 @@ def check_mode_log?(log)
     def ansible_module_message(log)
       msg_json = parsed_message_json(log)
       return _("Execution error: #{msg_json['msg']}") if msg_json['failed'].present?
+      return msg_json['censored'] if msg_json['censored'].present?
```

Comment:
I wonder if we should return the Ansible message, or write something ourself?

Reply:
I think the Ansible message is good. Do you want to ask someone for a third opinion?

Reply:
nah, if you're good with that, so am I :)